extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1040,11 +1101,34 @@ export class AmpA4A extends AMP.BaseElement {
     this.fromResumeCallback = false;
     this.experimentalNonAmpCreativeRenderMethod_ =
         platformFor(this.win).isIos() ? XORIGIN_MODE.SAFEFRAME : null;
+  }
+
+  /**
+   * Attempts to remove the current frame and free any associated resources.
+   * This function will no-op if this ad slot is currently in the process of
+   * being refreshed.
+   *
+   * @param {boolean=} force Forces the removal of the frame, even if
+   *   this.isRefreshing_ is true.
+   * @protected
+   */
+  destroyFrame(force = false) {
```

Comment:
No need to set default value as false as undefined is essentially equivalent here.